multiqc/modules/prokka/prokka.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from Prokka """
+
+from __future__ import print_function
+from collections import OrderedDict
+import json
+import logging
+import os
+import re
+
+from multiqc import config, BaseMultiqcModule, plots
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+class MultiqcModule(BaseMultiqcModule):
+
+    def __init__(self):
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='Prokka', anchor='prokka',
+        href='http://www.vicbioinformatics.com/software.prokka.shtml', 
+        info="is a software tool for the rapid annotation of prokaryotic genomes.")
+        
+        # Parse logs
+        self.prokka = dict()
+        for f in self.find_log_files(config.sp['prokka'], filehandles=True):
+            self.parse_prokka(f)
+
+        if len(self.prokka) == 0:
+            log.debug("Could not find any Prokka data in {}".format(config.analysis_dir))
+            raise UserWarning
+        
+        log.info("Found {} logs".format(len(self.prokka)))
+        self.write_data_file(self.prokka, 'multiqc_prokka')
+        
+        # Add Prokka's annotation stats to the general stats table
+        headers = OrderedDict()
+        headers['organism'] = {
+            'title': 'Organism',
+            'description': 'Organism',
+        }
+        #headers['contigs'] = {
+        #    'title': 'Contigs',
+        #    'description': 'Number of contigs',
+        #    "min": 0,
+        #}
+        headers['bases'] = {
+            'title': 'Bases',
+            'description': 'Number of bases',
+            "min": 0,
+            'format': '{:i}%'
+        }
+        headers['CDS'] = {
+            'title': 'CDS',
+            'description': 'Number of CDS',
+            "min": 0,
+            'format': '{:i}%'
+        }
+        headers['tRNA'] = {
+            'title': 'tRNA',
+            'description': 'Number of tRNA',
+            "min": 0,
+            'format': '{:i}%'
+        }
+        headers['rRNA'] = {
+            'title': 'rRNA',
+            'description': 'Number of rRNA',
+            "min": 0,
+            'format': '{:i}%'
```

Comment:
Ah.. It could be that if you use the `shared_key` value then it overrides the colour scheme and shares it across all columns with the same shared key? Maybe? It was a while ago that I wrote that code.